cpmonitor/models.py

```diff
@@ -136,6 +134,63 @@ def validate_unique(self, exclude=None):
             raise ValidationError(msgs)
 
 
+class CapChecklist(models.Model):
+    city = models.OneToOneField(
+        City,
+        models.PROTECT,
+        name="KAP Checkliste",
```

Comment:
I think, this `name` is really confusing, as can be seen in the JSON.

Reply:
@mdrie good catch, thanks

When I reviewed I was under the impression that this was so label for the admin gui. However, turns out this names the actual database field in which case you're totally right, it's confusing if not broken.

I have little experience with django and django admin, so I can't say I knew what I was doing, but I attempted a fix in #173 , please review.